dozer-types/src/types/field.rs

```diff
@@ -600,8 +601,8 @@ pub enum FieldType {
     Timestamp,
     /// Allows for every date from Jan 1, 262145 BCE to Dec 31, 262143 CE.
     Date,
-    /// Bytes that are valid BSON.
-    Bson,
+    /// Bytes that are valid JSON.
```

Comment:
We're not storing bytes, the the comment can just be `JsonValue`

Reply:
okok